figures/classes/Parallelogram.class.ts

```diff
@@ -0,0 +1,23 @@
+import { Rectangle } from './Rectangle.class';
+
+export class Parallelogram extends Rectangle {
+  private angle: number;
+
+  set angleDegrees(degrees) {
+    this.angle = degrees;
+  }
+
+  constructor(width: number, height: number, angle: number) {
```

Comment:
It is good,
Just to remind, you can use such syntax
`constructor(..., private angle: number) {` to avoid a separate declaration of angle (`private angle: number; // outside the constructor`)

Reply:
Ok, thanks!